INFO     log:test_07_delete_user.py:19 *************** 开始执行用例 ***************
INFO     log:http_client.py:40 接口请求地址 ==>> http://127.0.0.1/user/deleteUser
INFO     log:http_client.py:41 接口请求方式 ==>> POST
INFO     log:http_client.py:43 接口请求头 ==>> {
    "Content-Type": "application/json",
    "token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJyYmFjLXVzZXIiLCJpc3MiOiJsb25nY2hhbyIsImlhdCI6MTczMDcxNjQ2NCwiZXhwIjoxNzMwNzIwMDY0LCJpZCI6MTg0NjgwMTQ0MjQzMzY4MzQ1OH0.GMrZCqRE5ctBxmcLknWCE_QfxtQsOJpkrlywkAQXHwM"
}
INFO     log:http_client.py:45 接口请求 params 参数 ==>> {
    "userId": "1846801442433683458"
}
INFO     log:index_user_api.py:15 接口实际返回的结果为==>>{
    "success": false,
    "code": 9992,
    "message": "你没有权限访问该接口！",
    "time": 1730716468896,
    "path": null,
    "timestamp": null,
    "msg": null,
    "trackId": null
}
INFO     log:test_07_delete_user.py:29 *************** 恢复被删除用例状态 ***************
INFO     log:conftest.py:29 code ==>> 期望结果：9991， 实际结果：9992
INFO     log:conftest.py:30 返回信息 ==>> 期望结果：token过期失效！， 实际结果：你没有权限访问该接口！